src/includes/class-alma-wc-refund-helper.php

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * Alma Alma refund helper
+ *
+ * @package Alma_WooCommerce_Gateway
+ */
+
+use Alma\API\RequestError;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Not allowed' ); // Exit if accessed directly.
+}
+
+/**
+ * Alma_WC_Refund_Helper
+ */
+class Alma_WC_Refund_Helper {
+
+	/**
+	 * Logger
+	 *
+	 * @var Alma_WC_Logger
+	 */
+	private $logger;
+
+	/**
+	 * __construct.
+	 */
+	public function __construct() {
+		$this->logger = new Alma_WC_Logger();
+	}
+
+	/**
+	 * Gets the amount to refund.
+	 *
+	 * @param integer $refund_id Refund id.
+	 * @param bool    $display Tells if amount is supposed to be used for calculation or display.
+	 * @return int|string
+	 */
+	public function get_amount_to_refund( $refund_id, $display = false ) {
+		$refund           = new WC_Order_Refund( $refund_id );
+		$amount_to_refund = alma_wc_price_to_cents( floatval( $refund->get_amount() ) );
+		if ( true === $display ) {
+			$amount_to_refund = $refund->get_amount() . ' ' . $refund->get_currency();
+		}
+		return $amount_to_refund;
+	}
+
+	/**
+	 * Gets the comment of a refund (which is optional).
+	 *
+	 * @param integer $refund_id Refund id.
+	 * @return string
+	 */
+	public function get_refund_comment( $refund_id ) {
+		$refund = new WC_Order_Refund( $refund_id );
```

Comment:
you can pass WC_Order_Refund directly as parameter instead of instanciate it every where in you methods

So, this method seems useless

Reply:
Done.